src/components/dialog/__spec__.js

```diff
@@ -26,6 +26,16 @@ describe('Dialog', () => {
           instance.componentDidMount();
           expect(instance.centerDialog).toHaveBeenCalled();
         });
+
+        it('focuses on the dialog', () => {
+          spyOn(Dialog.prototype, 'componentDidMount').and.callThrough();
+          spyOn(Dialog.prototype, 'focusDialog');
+          const wrapper = mount(
+            <Dialog open={ true } onCancel={ onCancel } />
+          );
+          expect(Dialog.prototype.componentDidMount).toHaveBeenCalled();
```

Comment:
[minor] is this necessary? It's part of the react API - what I really care about is that the focus happens.
Thoughts?